versions.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.4.0, < 2.0.0"
 
   required_providers {
-    aws = ">= 3.0"
+    aws = ">= 4.3.5"
```

Comment:
Was there a version 4.3.5 of the AWS provider? I'm not seeing one in the AWS provider repo, so I'm wondering if this was a typo.